### DISCUSSION TITLE
Support filtering list by type

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#36317] [Other Feature Feedback, Questions, & Ideas]
I have at least one private repository starred, and quite a few public repositories. I'd like to be able to filter them at least as much as I can filter repositories in an org or user repositories list.

To be clear, these are the entrypoints I know consider to be a part of **Lists**:
* https://github.com/stars
* https://github.com/stars/jsoref/lists/to-check-spelling

**Comment by Belikeoscar** [comment]
Hi you can try sorting by type and choose the option for Private , that works for me.

**Reply by jsoref** [reply]
Please provide a picture (include the url) -- feel free to censor the actual repository names.
